recipes/libxslt/all/conanfile.py

```diff
@@ -0,0 +1,177 @@
+            debug = "yes" if self.settings.build_type == "Debug" else "no"
+            static = "no" if self.options.shared else "yes"
+
+            with tools.vcvars(self.settings):
+                args = ["cscript",
+                        "configure.js",
+                        "compiler=msvc",
+                        "prefix=%s" % self.package_folder,
+                        "cruntime=/%s" % self.settings.compiler.runtime,
+                        "debug=%s" % debug,
+                        "static=%s" % static,
+                        'include="%s"' % ";".join(self.deps_cpp_info.include_paths),
+                        'lib="%s"' % ";".join(self.deps_cpp_info.lib_paths),
+                        'iconv=no',
+                        'xslt_debug=no',
+                        'debugger=no',
+                        'crypto=no']
+                configure_command = ' '.join(args)
+                self.output.info(configure_command)
+                self.run(configure_command)
+
+                # Fix library names because they can be not just zlib.lib
+                def format_libs(package):
+                    libs = []
+                    for lib in self.deps_cpp_info[package].libs:
+                        libname = lib
+                        if not libname.endswith('.lib'):
+                            libname += '.lib'
+                        libs.append(libname)
+                    return ' '.join(libs)
+
+                def fix_library(option, package, old_libname):
+                    if option:
+                        tools.replace_in_file("Makefile.msvc",
+                                              "LIBS = %s" % old_libname,
+                                              "LIBS = %s" % format_libs(package))
+
+                if "icu" in self.deps_cpp_info.deps:
+                    fix_library(True, 'icu', 'wsock32.lib')
+
+                tools.replace_in_file("Makefile.msvc", "libxml2.lib", format_libs("libxml2"))
+                tools.replace_in_file("Makefile.msvc", "libxml2_a.lib", format_libs("libxml2"))
+
+                with tools.environment_append(VisualStudioBuildEnvironment(self).vars):
+                    self.run("nmake /f Makefile.msvc install")
+
+    def _build_with_configure(self):
+        in_win = self.settings.os == "Windows"
+        env_build = AutoToolsBuildEnvironment(self, win_bash=in_win)
+        if not in_win:
+            env_build.fpic = self.options.fPIC
+        full_install_subfolder = tools.unix_path(self.package_folder) if in_win else self.package_folder
+        # fix rpath
+        if self.settings.os == "Macos":
+            tools.replace_in_file(os.path.join(self._full_source_subfolder, "configure"), r"-install_name \$rpath/", "-install_name ")
+        configure_args = ['--with-python=no', '--prefix=%s' % full_install_subfolder]
+        if env_build.fpic:
+            configure_args.extend(['--with-pic'])
```

Comment:
I think the autotools helper handles this
```suggestion
        if env_build.fpic:
            configure_args.extend(['--with-pic'])
```

Reply:
Mmm, I fail to see the difference. Do my eyes betray me ?

Reply:
Haha, I meant to remove the two lines :smile: